GlobalPlanner: 
  default_tolerance: 0.1
  use_dijkstra: false
  use_grid_path: true